src/main/java/life/qbic/portal/portlet/connection/ConnectionHandler.java

```diff
@@ -53,7 +53,7 @@ public OpenBisConnection connectToOpenBis() {
 
       // get a reference to AS API
       IApplicationServerApi app = HttpInvokerUtils.createServiceStub(IApplicationServerApi.class,
-          openBisUrl + IApplicationServerApi.SERVICE_URL,
+          openBisUrl + "/openbis/openbis" + IApplicationServerApi.SERVICE_URL,
```

Comment:
Should `/openbis/openbis` also be part of the `openBisUrl` ?

Reply:
I also thought so but it seems not to be like this. Feel free to test it on your system. For me, it only works with the addition of it